mongodbatlas/resource_mongodbatlas_network_container_test.go

```diff
@@ -15,19 +15,21 @@ import (
 
 func TestAccNetworkRSNetworkContainer_basicAWS(t *testing.T) {
 	var (
-		container        matlas.Container
-		randInt          = acctest.RandIntRange(0, 255)
-		randIntUpdated   = acctest.RandIntRange(0, 255)
-		resourceName     = "mongodbatlas_network_container.test"
-		cidrBlock        = fmt.Sprintf("10.8.%d.0/24", randInt)
-		cidrBlockUpdated = fmt.Sprintf("10.8.%d.0/24", randIntUpdated)
-		providerName     = "AWS"
-		orgID            = os.Getenv("MONGODB_ATLAS_ORG_ID")
-		projectName      = acctest.RandomWithPrefix("test-acc")
+		container                matlas.Container
+		randInt                  = acctest.RandIntRange(0, 255)
+		randIntUpdated           = acctest.RandIntRange(0, 255)
+		resourceName             = "mongodbatlas_network_container.test"
+		dataSourceName           = "data.mongodbatlas_network_container.test"
+		dataSourceContainersName = "data.mongodbatlas_network_containers.test"
+		cidrBlock                = fmt.Sprintf("10.8.%d.0/24", randInt)
+		cidrBlockUpdated         = fmt.Sprintf("10.8.%d.0/24", randIntUpdated)
+		providerName             = "AWS"
+		orgID                    = os.Getenv("MONGODB_ATLAS_ORG_ID")
+		projectName              = acctest.RandomWithPrefix("test-acc")
 	)
 
-	resource.Test(t, resource.TestCase{
-		PreCheck:          func() { testAccPreCheck(t) },
+	resource.ParallelTest(t, resource.TestCase{
```

Comment:
I still can't get over how simple a change this is vs the impact it has. Great work!